src/main/java/persistence/sql/meta/ColumnFields.java

```diff
@@ -0,0 +1,54 @@
+package persistence.sql.meta;
+
+import org.h2.util.StringUtils;
+
+import java.lang.reflect.Field;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ColumnFields {
+
+    private List<ColumnField> columnFields;
+
+    public ColumnFields(Class<?> clazz) {
+        this.columnFields = extract(clazz);
+    }
+
+    private List<ColumnField> extract(Class<?> clazz) {
+        return Arrays.stream(clazz.getDeclaredFields()).map(ColumnField::new)
+                .filter(ColumnField::isNotTransient).collect(Collectors.toList());
+    }
```

Comment:
```suggestion
    private List<ColumnField> extract(Class<?> clazz) {
        return Arrays.stream(clazz.getDeclaredFields())
                .filter(field -> !field.isAnnotationPresent(Transient.class))
                .map(ColumnField::new)
                .collect(Collectors.toList());
    }
```

- 개인적으로 Stream 을 쓸땐 체이닝 별로 내려써주시면 읽기 좀 더 편하다고 느껴요 :)
- 만들때 처음부터 Transient 를 제거한다면, 필요없는 객체를 생성하는 비용을 아낄 수 있지 않을까요?